partition_table.go

```diff
@@ -424,43 +402,27 @@ func (p *PartitionTable) markRecovered(ctx context.Context) error {
 	}
 }
 
-func (p *PartitionTable) handleConsumerErrors(ctx context.Context, errs *multierr.Errors, cons sarama.PartitionConsumer) {
-	for {
-		select {
-		case consError, ok := <-cons.Errors():
-			if !ok {
-				return
-			}
-			err := fmt.Errorf("Consumer error: %v", consError)
-			p.log.Printf("%v", err)
-			errs.Collect(err)
-			// if there's an error, close the consumer
-			cons.AsyncClose()
-		case <-ctx.Done():
-			return
-		}
-	}
-}
-
-func (p *PartitionTable) drainConsumer(cons sarama.PartitionConsumer, errs *multierr.Errors) {
+func (p *PartitionTable) drainConsumer(cons sarama.PartitionConsumer) error {
 
-	ctx, cancel := context.WithTimeout(context.Background(), 10*time.Second)
+	timeoutCtx, cancel := context.WithTimeout(context.Background(), 1*time.Second)
```

Comment:
You changed the time constant. Maybe give it a name and list it as constant in the top of the file.

Reply:
yeah because I thought draining the channels into the void shouldn't take 10 seconds :). But good point, let's put it up as a constant.

Reply:
done.